frontend/ui/data/creoptions.lua

```diff
@@ -537,19 +567,19 @@ Note that your selected font size is not affected by this setting.]]),
                 },
                 help_text = _([[Set the font weight delta from "regular" to apply to all fonts.
 
-- 0 will use the "regular (400)" variation of a font.
-- +1 will use the "medium (500)" variation of a font if available
-- +3 will use the "bold (700)" variation of a font if available
-If a font variation is not available, as well as for fractional adjustments, a font variation will be synthesized from the nearest available weight of the font.]]),
+- 0 will use the "Regular (400)" variation of a font.
+- +1 will use the "Medium (500)" variation of a font if available.
+- +3 will use the "Bold (700)" variation of a font if available.
+If a font variation is not available, as well as for fractional adjustments, it will be synthesized from the nearest available weight.]]),
                 help_text_func = function(configurable, document)
                     local font_face = document:getFontFace()
-                    local available_weights = cre.getFontFaceAvailableWeights(font_face)
-                    return T(_("The default font '%1' is available in %2."), font_face, table.concat(available_weights, ", "))
+                    local available_weights = prettifyCreWeights(cre.getFontFaceAvailableWeights(font_face))
+                    return T(_("The default font '%1' provides the following weight classes: %2."), font_face, table.concat(available_weights, ", "))
```

Comment:
I think in Chinese they might prefer the full-width comma, though hopefully it matters little. It's not something you'd be looking at every day.

Reply:
As the concat token, you mean?

That'd be tricky to make CJK-only, and would look ugly in LGC otherwise, no?

Reply:
It's a semi-random specific example for a more generic remark.

> That'd be tricky to

Er, that's why we have our gettext implementation in the first place? ;-)

Reply:
Oh, right, you meant simply by making it translatable :D.